### DISCUSSION TITLE
How to position two images next to each other?

### DISCUSSION THREAD

**Discussion by wielandb** [zauberzeug/nicegui#485] [Q&A] [answered]
The code 

```py
from nicegui import ui

with ui.row():
    ui.image("https://picsum.photos/id/377/640/360")
    ui.button("Button 1")
    ui.button("Button 2")
    ui.button("Button 3")
    ui.image("https://picsum.photos/id/377/640/360")

ui.run()
```

does not, as would one expect, position all theese elements next to each other, but always forces a line break for images:

![image](https://user-images.githubusercontent.com/29458682/224025881-941a936f-a14a-4d25-9085-fcdb69052020.png)

Is this behaviour intended? Is it a technical limitation? If not, how to position two images next to each other?

**Comment by falkoschindler** [comment, marked as answer]
You can add the class "no-wrap" to the row. Because images get their size automatically from the surrounding container, we either need to give the row or the images themselves a width or height:

```py
with ui.row().classes('no-wrap'):
    ui.image("https://picsum.photos/id/377/640/360").classes('w-60')
    ui.button("Button 1")
    ui.button("Button 2")
    ui.button("Button 3")
    ui.image("https://picsum.photos/id/377/640/360").classes('w-60')
```

**Reply by wielandb** [reply]
As I am not super proficient with styling and don't know if this is expected or normal when working in webdesign, I am unsure if maybe it would be a good idea to reference this behaviour in the documentation?

**Reply by rodja** [reply]
Thanks! We are internally working on a simplified styling solution for non-web-devs in #117.
